integration-tests/build.gradle.kts

```diff
@@ -18,6 +18,7 @@ dependencies {
   implementation(libs.java.stellar.sdk)
   implementation(libs.google.gson)
   implementation(libs.okhttp3)
+  implementation(libs.log4j2.api)
```

Comment:
`ServiceLoaderUtil` was moved from `log4j2.core` to `log4j2.api` in version `2.19.0`